libs/model/src/subscription/UpdateSubscriptionPreferences.command.ts

```diff
@@ -0,0 +1,23 @@
+import { schemas, type Command } from '@hicommonwealth/core';
+import { models } from '../database';
+
+export const UpdateSubscriptionPreferences: Command<
+  typeof schemas.commands.UpdateSubscriptionPreferences
+> = () => ({
+  ...schemas.commands.UpdateSubscriptionPreferences,
+  auth: [],
+  secure: true,
+  body: async ({ payload, actor }) => {
+    const { 1: rows } = await models.SubscriptionPreference.update(payload, {
```

Comment:
guess upsert doesn't work here?

Reply:
Technically unnecessary since there is a trigger which creates the SubscriptionPreference when a User is created and if a user updates their preference that assumes it already exists and can be viewed on the client.

Reply:
The problem is we currently don't strictly adhere to DDD and so Users can be created outside the Sequelize flow -> for example in the Discourse imports we don't use the Sequelize models which would mean new users would not have preferences when imported via Discourse.

I 100% agree it would be better to use Sequelize hooks for all triggers but it isn't an option for some of our models which have custom queries in which the hooks are not respected.